### PR TITLE
docs: improve guides

### PR DESCRIPTION
**Pull request by wizardofhogwarts**

### CODE REVIEW

**Comment by wizardofhogwarts** [comment]
@montogeek @EugeneHlushko Please review

**Comment by wizardofhogwarts** [comment]
@EugeneHlushko and @montogeek please review

File: src/content/guides/asset-management.md [hunk 1]
```diff
@@ -491,7 +492,7 @@ T> This can be especially helpful when implementing some sort of data visualizat
 
 ## Global Assets
 
-The coolest part of everything mentioned above, is that loading assets this way allows you to group modules and assets together in a more intuitive way. Instead of relying on a global `/assets` directory that contains everything, you can group assets with the code that uses them. For example, a structure like this can be very useful:
+The coolest part of everything mentioned above is that loading asset this way allows you to group modules and assets together in a more intuitive way. Instead of relying on a global `/assets` directory that contains everything, you can group assets with the code that uses them. For example, a structure like this can be very useful:
```
Comment (EugeneHlushko) [inline src/content/guides/asset-management.md]: I think it does speak of assets

File: src/content/guides/asset-management.md [hunk 2]
```diff
@@ -146,7 +147,7 @@ Entrypoint main = bundle.js
 ...
 ```
 
-Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. To see what webpack did, inspect the page (don't view the page source, as it won't show you the result) and look at the page's head tags. It should contain our style block that we imported in `index.js`.
+Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. To see what webpack did, inspect the page (don't view the page source, as it won't show you the result) and look at the page's head tags. It should contain the style block that we imported in `index.js`.
```
Comment (EugeneHlushko) [inline src/content/guides/asset-management.md]: is this necessary? I think they've used `our` to keep speech style consistent

File: src/content/guides/author-libraries.md [hunk 3]
```diff
@@ -99,15 +100,15 @@ import * as webpackNumbers from 'webpack-numbers';
 webpackNumbers.wordToNum('Two');
 ```
 
-- __CommonJS module require:__
+- __CommonJS module requires:__
```
Comment (EugeneHlushko) [inline src/content/guides/author-libraries.md]: that is a header for example require call of `CommonJS ` module, why is this change necessary?

File: src/content/guides/author-libraries.md [hunk 4]
```diff
@@ -99,15 +100,15 @@ import * as webpackNumbers from 'webpack-numbers';
...
 
 ``` js
 const webpackNumbers = require('webpack-numbers');
 // ...
 webpackNumbers.wordToNum('Two');
 ```
 
-- __AMD module require:__
+- __AMD module requires:__
```
Comment (EugeneHlushko) [inline src/content/guides/author-libraries.md]: that is a header for example `require` call of `AMD ` module, why is this change necessary?

File: src/content/guides/author-libraries.md [hunk 5]
```diff
@@ -264,7 +265,7 @@ __webpack.config.js__
 
 T> Note that the `library` setup is tied to the `entry` configuration. For most libraries, specifying a single entry point is sufficient. While [multi-part libraries](https://github.com/webpack/webpack/tree/master/examples/multi-part-library) are possible, it is simpler to expose partial exports through an [index script](https://stackoverflow.com/questions/34072598/es6-exporting-importing-in-index-file) that serves as a single entry point. Using an `array` as an `entry` point for a library is __not recommended__.
 
-This exposes your library bundle available as a global variable named `webpackNumbers` when imported. To make the library compatible with other environments, add `libraryTarget` property to the config. This will add the different options about how the library can be exposed.
+This exposes your library bundle available as a global variable named `webpackNumbers` when imported. To make the library compatible with other environments, add `libraryTarget` property to the config. This will add different options about how the library can be exposed.
```
Comment (EugeneHlushko) [inline src/content/guides/author-libraries.md]: Maybe go with `various` here?
```suggestion
This exposes your library bundle available as a global variable named `webpackNumbers` when imported. To make the library compatible with other environments, add `libraryTarget` property to the config. This will add various options about how the library can be exposed.
```

File: src/content/guides/author-libraries.md [hunk 6]
```diff
@@ -305,7 +306,7 @@ W> With webpack 3.5.5, using `libraryTarget: { root:'_' }` doesn't work properly
 
 ### Final Steps
 
-Optimize your output for production by following the steps in the [production guide](/guides/production). Let's also add the path to your generated bundle as the package's `main` field in with our `package.json`
+Optimize your output for production by the following steps in the [production guide](/guides/production). Let's also add the path to your generated bundle as the package's `main` field in with the `package.json`
```
Comment (EugeneHlushko) [inline src/content/guides/author-libraries.md]: i think it was correct. 
- By Following what?
- the steps

Comment (EugeneHlushko) [reply src/content/guides/author-libraries.md]: Same about `our`, this is article's style no?

File: src/content/guides/production.md [hunk 7]
```diff
@@ -18,16 +18,17 @@ contributors:
...
 
 T> This walkthrough stems from [Tree Shaking](/guides/tree-shaking) and [Development](/guides/development). Please ensure you are familiar with the concepts/setup introduced in those guides before continuing on.
 
 
 ## Setup
 
-The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
+The goals of _development_ and _production_ build differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: one and two (builds), why this change?

File: src/content/guides/production.md [hunk 8]
```diff
@@ -206,7 +207,7 @@ If you decide to try another minification plugin, just make sure your new choice
 
 ## Source Mapping
 
-We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in _production_ as opposed to the `inline-source-map` we used in _development_:
+We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in _production_ as opposed to the `inline-source-map` we used in the _development_:
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: so preceding `in production` looks good to you, but in development should receive a `the` ? I don't get it

File: src/content/guides/production.md [hunk 9]
```diff
@@ -230,6 +231,6 @@ It is crucial to minimize your CSS on production, please see [Minimizing for Pro
 
 ## CLI Alternatives
 
-Some of what has been described above is also achievable via the command line. For example, the `--optimize-minimize` flag will include the `TerserPlugin` behind the scenes. The `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above. And, `webpack -p` will automatically invoke both those flags and thus the plugins to be included.
+Some of what has been described above are also achievable via the command line. For example, the `--optimize-minimize` flag will include the `TerserPlugin` behind the scenes. The `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above. And, `webpack -p` will automatically invoke both those flags and thus the plugins to be included.
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]:
```suggestion
Some of what has been described above can also be achieved by using the command line. For example, the `--optimize-minimize` flag will include the `TerserPlugin` behind the scenes. The `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above. And, `webpack -p` will automatically invoke both those flags and thus the plugins to be included.
```

File: src/content/guides/production.md (hunk 8 above)
Comment (wizardofhogwarts) [reply src/content/guides/production.md]: @EugeneHlushko I must have missed it while reading. Adding in next commit.

File: src/content/guides/production.md (hunk 7 above)
Comment (wizardofhogwarts) [reply src/content/guides/production.md]: According to subject verb agreement, `build` will fit into this context.

File: src/content/guides/author-libraries.md (hunk 6 above)
Comment (wizardofhogwarts) [reply src/content/guides/author-libraries.md]: > i think it was correct.
> 
> * By Following what?
> * the steps

I misunderstood may be you are right.

Comment (wizardofhogwarts) [reply src/content/guides/author-libraries.md]: > Same about `our`, this is article's style no?

IMO `our` isn't a good choice in formal documentation.

File: src/content/guides/production.md (hunk 7 above)
Comment (wizardofhogwarts) [reply src/content/guides/production.md]: @EugeneHlushko @montogeek please check it again

File: src/content/guides/author-libraries.md (hunk 6 above)
Comment (wizardofhogwarts) [reply src/content/guides/author-libraries.md]: @EugeneHlushko @montogeek please check it again

**Comment by montogeek** [comment]
Any update?

**Comment by wizardofhogwarts** [comment]
> Any update?

According to me, it's fine.

**Comment by wizardofhogwarts** [comment]
@dhruvdutt They are already

**Comment by montogeek** [comment]
GitHub shows conflicts for me
image

**Comment by wizardofhogwarts** [comment]
> GitHub shows conflicts for me
> image

Just fixed

File: src/content/guides/production.md [hunk 10]
```diff
@@ -18,17 +18,18 @@ contributors:
...
 
 T> This walkthrough stems from [Tree Shaking](/guides/tree-shaking) and [Development](/guides/development). Please ensure you are familiar with the concepts/setup introduced in those guides before continuing on.
 
 
 ## Setup
 
-The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
+The goals of _development_ and _production_ build differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: I still think that `development build and production build` becomes `builds`.
If it was saying: `the difference between those builds` you wouldn't change it to `build` right?

I did take a look at this: https://www.grammarbook.com/grammar/subjectVerbAgree.asp
But still cannot see why you want to change that 😅

Comment (wizardofhogwarts) [reply src/content/guides/production.md]: I was talking in context of goals. But no worries if you cross checked it. I may revert it if u say.

Comment (EugeneHlushko) [reply src/content/guides/production.md]: or easier apply suggest 
```suggestion
The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```

**Comment by wizardofhogwarts** [comment]
@montogeek

**Comment by EugeneHlushko** [comment]
Thanks!
